### PR TITLE
Game options

### PR DESCRIPTION
**Pull request by Eccb7**
In this pull request, I have added the game options

### CODE REVIEW

**Review by Bleu-Yves-Sopoude — CHANGES_REQUESTED** [review]
Great job implementing the features but there is still something you should do to complete the task.

You have not implemented the options for displaying the game or it is just not working. Kindly fix it and I'll approve.

**Comment by Eccb7** [comment]
Hi @Bleu-Yves-Sopoude 
Well, I can add a game plus display it's list

File: .rubocop.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ AllCops:
 Layout/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Max: 20
+  Max: 40
```
Comment (henokkhm) [inline .rubocop.yml:13]: Please revert this change since we have been working with these settings since the start of the project.

File: classes/game.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@ def initialize(title, multiplayer, last_played_at, publish_date, archived)
     @title = title
     @multiplayer = multiplayer
     @last_played_at = last_played_at
+    @author = nil
```
Comment (henokkhm) [inline classes/game.rb:9]: There is no need to set the **author** to **nil** here because it is already set to null by the initialize method of Item, and you are calling that on line 5

File: lib/console_manager.rb [hunk 3]
```diff
@@ -124,8 +126,53 @@ def list_all_music
...
+
+    # Parse dates into Date objects
+    last_played_at = Date.parse(last_played_date)
+    publish_date = Date.parse(publish_date)
+
+    print 'Archived (Y/N): '
+    archived = gets.chomp.upcase
+
+    # Action
+    @game_manager.add_game(title, multiplayer == 'Y', last_played_at, publish_date, archived == 'Y')
```
Comment (henokkhm) [inline lib/console_manager.rb:157]: I think you should give the user the option to add an **Author** to the **Game** they just created. Currently, there is no way a user can add an author. You can refer to my implementation that lets the user add a **Label** to a **Book**.

File: lib/game_manager.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  attr_reader :games_list
+
+  def add_game(title, multiplayer, last_played_at, publish_date, archived)
+    game = Game.new(title, multiplayer, last_played_at, publish_date, archived)
+    @games_list.push(game)
+    game
+  end
+
+  def authors_list
+    authors
```
Comment (henokkhm) [inline lib/game_manager.rb:17]: This **authors** variable is not defined anywhere. Hence, the following error is thrown when this line executes.
image

Instead of putting the Authors in the GameManager, you can create an **AuthorsManager** class that works in the same way.

**Review by henokkhm — CHANGES_REQUESTED** [review]
Hi @Eccb7 

Good job so far. I can create Games and display the list of Games without any problems. 
However, you have not provided the user with a way to add Authors to the Games they create. Further, the List Authors functionality, i.e. option 11 in the main menu, throws the following error:
image


 Please fix this issue as soon as possible so that I can approve your PR. Thanks

File: classes/game.rb (hunk 2 above)
Comment (Eccb7) [reply classes/game.rb:9]: Actually, it's this that sorted the errors from my end

**Review by henokkhm — APPROVED** [review]
Hi @Eccb7 

## Status: Approved :tada: 

Thank you for making the changes. You can now merge your code.

**Review by Bleu-Yves-Sopoude — APPROVED** [review]
Great Job
